examples/ComputeBoids.cpp

```diff
@@ -223,10 +223,11 @@ void initSim() {
         }
     )");
 
-    nxt::BindGroupLayout bgl = device.CreateBindGroupLayoutBuilder()
-        .SetBindingsType(nxt::ShaderStageBit::Compute, nxt::BindingType::UniformBuffer, 0, 1)
-        .SetBindingsType(nxt::ShaderStageBit::Compute, nxt::BindingType::StorageBuffer, 1, 2)
-        .GetResult();
+    auto bgl = utils::MakeBindGroupLayout(device, {
+        { 0, nxt::ShaderStageBit::Compute, nxt::BindingType::UniformBuffer, 1 },
+        { 1, nxt::ShaderStageBit::Compute, nxt::BindingType::StorageBuffer, 1 },
+        { 2, nxt::ShaderStageBit::Compute, nxt::BindingType::StorageBuffer, 1 },
```

Comment:
Just for my own edification: why not use a single entry with an array size of two here, as it was before, instead of two entries? Is the array size semantically different from the old "count"?